js/apps/admin-ui/src/client-scopes/details/ScopeForm.tsx

```diff
@@ -55,6 +57,15 @@ export const ScopeForm = ({ clientScope, save }: ScopeFormProps) => {
     defaultValue: "false",
   });
 
+  const selectedProtocol = useWatch({
+    control,
+    name: "protocol",
+    defaultValue: clientScope?.protocol ?? providers[0],
+  });
+
+  const isOid4vcProtocol = () => selectedProtocol === OID4VC_PROTOCOL;
+  const isOid4vcEnabled = () => isFeatureEnabled(Feature.OpenId4VCI);
```

Comment:
There should be no need to defer the computation of these values, they can be assigned to a variable in the scope immediately:
```suggestion
  const isOid4vcProtocol = selectedProtocol === OID4VC_PROTOCOL;
  const isOid4vcEnabled = isFeatureEnabled(Feature.OpenId4VCI);
```